_includes/footer.html

```diff
@@ -1,3 +1,7 @@
+<div class="edit">
+  <a href="{{site.github.repository_url}}/edit/main/{{page.path}}">✎ Edit this page</a>
```

Comment:
We got build issue based of that value missing
https://github.com/redhat-developer/app-services-architecture/runs/4887126419?check_suite_focus=true

Reply:
actually that is other issue. Let me fix that